model/src/main/kotlin/OrtResult.kt

```diff
@@ -76,18 +79,27 @@ data class OrtResult(
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val data: CustomData = mutableMapOf()
 
-    private val excludedProjects: Set<Identifier> by lazy {
-        getProjects()
-            .filter { project ->
+    private data class ProjectEntry(val project: Project, val isExcluded: Boolean)
+
+    private val projects: Map<Identifier, ProjectEntry> by lazy {
+        log.info { "Computing excluded projects..." }
```

Comment:
I think that should be a debug message.

Reply:
I prefer to keep it on info level as I want to make sure this is printed, because
the functions might take a while and the log reader can see what is going on.

Reply:
If this is about duration the first message should include something like "(this can take several minutes)".

Progress output would be even better, like "100/2000 done", but that can be done separately.

Reply:
the message now indicates it may take a while.